ci/testnet-manager.sh

```diff
@@ -444,6 +459,26 @@ deploy() {
       fi
     )
     ;;
+  testnet-tds)
+    (
+      set -x
+      # TODO: Should we spread the few nodes around multiple zones?
+      # shellcheck disable=SC2068
+      # shellcheck disable=SC2086
+      NO_LEDGER_VERIFY=1 \
+      NO_VALIDATOR_SANITY=1 \
+        ci/testnet-deploy.sh -p tds-solana-com -C gce ${GCE_ZONE_ARGS[0]} \
+          -t "$CHANNEL_OR_TAG" -n "$GCE_NODE_COUNT" -c 1 -P -u \
+          -a tds-solana-com --hashes-per-tick auto \
+          ${skipCreate:+-e} \
+          ${skipStart:+-s} \
+          ${maybeStop:+-S} \
+          ${maybeDelete:+-D} \
+          -k 1000 -K /tmp/stakes.yml \
```

Comment:
Where does `/tmp/stakes.yml` come from?

Reply:
Currently it's a file sitting on m-s-c VM with some a dummy pubkey: stake data I created.  This path can point to anything that contains our external stake holders.  I think a cleaner option might be to wget this file from some managed location onto the machine running testnet-manager.sh, then pass it down from there.  I was planning to add that feature to a subsequent PR unless you think it is important to get in from the beginning here.

Reply:
sg, follow-up PR totally fine by me